mod/photos.php

```diff
@@ -1027,6 +1027,10 @@ function photos_content(App $a)
 
 	// Display a single photo album
 	if ($datatype === 'album') {
+		// if $datum is not a valid hex, redirect to the default page
+		if (!Strings::isHex($datum)) {
+			$a->internalRedirect();
```

Comment:
Can you please redirect to `/photos` instead?

Reply:
`/photos` without a parameter just returns the base-pagelayout with "no photos selected". That's why I thought it's better to redirect to the landing page

Reply:
Indeed, what about `/photos/{nickname}` instead?